librosa/core/notation.py

```diff
@@ -714,15 +733,15 @@ def key_to_notes(key: str, *, unicode: bool = True) -> List[str]:
     if multiple:
         sign_map = {+1: "♯", -1: "♭"}
         additional_acc = sign_map[np.sign(offset)]
-        intermediate_notes = key_to_notes(tonic+additional_acc*(abs(offset)-1)+':'+scale)
+        intermediate_notes = key_to_notes(tonic+additional_acc*(abs(offset)-1)+':'+scale, natural = natural)
         notes = [__simplify_note(note, additional_acc) for note in intermediate_notes]
```

Comment:
Thinking about this: do we need to propagate `natural=natural` here?  If it's immediately followed by a call to `__simplify_note`, wouldn't a natural marker be automatically purged?

It seems to me like we should be able to call through explicitly with `natural=False` to bypass unnecessary decorations, and only do one fixup at the top-level call (handled below).

Reply:
I think that's right. I implemented this change. 

For the mode implementation, though, we should pass through `natural = natural`, I believe.

Reply:
> For the mode implementation, though, we should pass through `natural = natural`, I believe.

Sorry, brain isn't parsing this for some reason.  What do you mean by "mode implementation"?

Reply:
When we call `key_to_notes` with parameter `key='G:loc'`, for example. The way this currently works is by simply returning `key_to_notes` with parameter `key='Ab:maj'`, so we need to pass the same value for `natural` through into this call.

Reply:
Ah, got it - thanks for clarifying!